### PR TITLE
APF defaults.go: use already defined catch-all name constant

### PR DESCRIPTION
**Pull request by adtac**
**What type of PR is this?**

/kind cleanup

Something minor I noticed when I was looking through defaults.go. I've confirmed that "catch-all" is not used anywhere else in the code (except comments).

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

/sig api-machinery
cc @MikeSpreitzer

### CODE REVIEW

**Comment by adtac** [comment]
/assign @lavalamp 

for API review

**Review by MikeSpreitzer — APPROVED** [review]
This is not even an API change, it is just changing the implementation to use symbolic constants instead of literals in a few places.

**Comment by adtac** [comment]
>This is not even an API change, it is just changing the implementation to use symbolic constants instead of literals in a few places.

I thought all PRs that had fejta-bot commenting about API review required an actual review from someone in the API approvers list

**Comment by MikeSpreitzer** [comment]
Sorry, my comment was not clear.  I was just emphasizing that although the automation identifies this PR as an API change, it is not "really" an API change.  I am not denying that the automation requires an API approval.

**Pull request merged**
